### PR TITLE
GOOGLE Chrome: attempt to install and run google chrome

### PR DESCRIPTION
**Pull request by DimStar77**
It's basically a copy of the chromium test, but with added installation step to get it from google directly

### CODE REVIEW

File: tests/x11/chrome.pm [hunk 1]
```diff
@@ -0,0 +1,36 @@
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    my $CHROME_URL = "https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm";
+
```
Comment (coolo) [inline tests/x11/chrome.pm]: You want to either exclude this test for !x86_64 or handle get_var('ARCH') here

File: tests/x11/chrome.pm [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    mouse_hide;
+
+    x11_start_program("xterm");
+    assert_screen('xterm-started');
+
+    script_sudo "zypper -n install $CHROME_URL";
+    save_screenshot;
```
Comment (coolo) [inline tests/x11/chrome.pm]: no handling of exit status? script_sudo only checks for idle machine - which is a bit weak

**Comment by sysrich** [comment]
Stupid question, but going to ask it anyway - do we care about testing chrome on SLES?

**Comment by coolo** [comment]
you ask the wrong audience :)

**Comment by sysrich** [comment]
Fine, I'll ask someone who cares

@sysrich do you think it's important to test Chrome on SLES?

@sysrich in an ideal world, yes, but what will SUSE do if it doesn't work?

@sysrich probably nothing, given it's prop. software and it's pretty much up to Google if they want Chrome to work on SLE

@sysrich so no point?

@sysrich agreed

**Comment by coolo** [comment]
the main reason we want it for leap is to see if it can be installed - e.g. we lacked a lsb package in leap. That is a valid test for SLE too IMO

**Pull request merged**